### DISCUSSION TITLE
Archive URL

### DISCUSSION THREAD

**Discussion by divadwong68** [SinTan1729/chhoto-url#30] [Ideas]
It was be nice to be able to archive a URL so that it's no longer accessible. And then have the option to then delete it or to Unarchive it if it was deleted accidentally.

**Comment by SinTan1729** [comment]
This will need a pretty substantial reworking of both the frontend and the backend. Frontend might be easier, but the backend will need either an extra database, or change of schema in the existing database (and hence I'll have to deal with automated migration etc.)

I'm not sure if it's worth the hassle since this project isn't meant to be a complete solution. For more features, projects like YOURLS are a better choice.

**Reply by divadwong68** [reply]
I checked out YOURLS and they don't have the feature to undo a delete either.  I understand it's quite complicated but just thought it would be a nice feature.  I understand you want to keep this super simple so I'll look elsewhere.  Thanks.
